common/lib/xmodule/xmodule/templates/html/image_modal.yaml

```diff
@@ -1,7 +1,23 @@
 ---
 metadata:
-    display_name: Full Screen Image
+    display_name: Full Screen Image Tool
 data: |
-        <h2>Full Screen Image</h2>
-        <p>Use the full screen image tool to allow students to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
-        <a href="http://static.class.stanford.edu/stanford-hills-big.jpg" class="modal-content"><img alt="The Stanford Hills" src="http://static.class.stanford.edu/stanford-hills-small.jpg" /></a>
\ No newline at end of file
+        <h2>Full Screen Image Tool</h2>
+        <p>Use the Full Screen Image tool to allow learners to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
+        <p>With the Full Screen Image tool, learners can see the image's details as well as its context within the unit.</p>
+        <p>You enable learners to view the image in the full screen by wrapping the image in a link to the larger version of the image. The HTML format required to use the Full Screen Image tool is:</p>
+        <pre>
+        &lt;a href="<i>path to full screen image</i>" class="modal-content"&gt;
+          &lt;img alt="<i>text for screen readers</i>" 
+          src="<i>path to image to include in unit page</i>"/&gt;
+        &lt;/a&gt;
+        </pre>
+        <p>Modify this template as needed. You can modify the example below for your own use.</p>
+        <ol>
+        <li>Replace the value of the link's <strong>href</strong> attribute with the path to the full size image. Do not change the value of the class attribute.</li>
+        <li>Replace the value of the image's <strong>src</strong> attribute with the path to the image that will appear in the unit before a learner opens it in full screen mode.</li>
+        <li>Replace the value of the image's <strong>alt</strong> attribute with text that both describes the image and the action or destination of clicking on the image. This text will be read by screen readers and is required for content accessibility.</li>
+        </ol>
```

Comment:
I really prefer this format for presenting an example and then describing each of the components that users can set in the example, to the way the iFrame example is presented above.